docs/packages-license.md

```diff
@@ -1908,6 +1908,17 @@ LGPL-3.0-or-later permitted
 
 
 
+<a name="@radix-ui/react-radio-group"></a>
+### @radix-ui/react-radio-group v1.2.2
+#### 
+
+##### Paths
+* /home/runner/work/liam/liam
+
+<a href="http://opensource.org/licenses/mit-license">MIT</a> permitted
+
+
+
```

Comment:
https://github.com/liam-hq/liam/pull/651/files#r1940402066  

Just to be safe, I'll suggest doing a rebase.

Reply:
I pushed 0d9c3ca as a substitute for f296701 . This commit doesn't include the changes related to `radix-ui/react-radio-group` that have already been merged into `main`.